docs/data-sources/cluster.md

```diff
@@ -322,6 +322,11 @@ Required:
 
 - `aws_availability_zone` (String) The Availability Zone where the AWS nodes are placed
 
+
```

Comment:
nit: remove these extra lines, they are auto-generated every time, so just check the changes and revert these.

Reply:
revert this changes